common/src/models/deployment.rs

```diff
@@ -150,18 +150,21 @@ pub fn get_deployments_table(deployments: &Vec<Response>, service_name: &str, pa
             ]);
         }
 
-        format!(
-            r#"
-Most recent {} for {}
-{}
-
-{}
-"#,
+        let formatted_table = format!(
+            "Most recent {} for {}\n{}",
             "deployments".bold(),
             service_name,
             table,
-            "More projects might be available on the next page using --page.".bold()
-        )
+        );
+
+        if deployments.len() == limit as usize {
```

Comment:
I like it! But I guess this will still be wrong when there are exactly `limit` projects, right? Could we change the limit logic to be `limit + 1`, then only display `limit` projects. So say there are 15 projects, we have a limit of 10 and we show those 10, but we got 11, so we know there are more projects on the next page.

Reply:
Oh, and a `cargo fmt` should fix CI

Reply:
> I like it! But I guess this will still be wrong when there are exactly `limit` projects, right? Could we change the limit logic to be `limit + 1`, then only display `limit` projects. So say there are 15 projects, we have a limit of 10 and we show those 10, but we got 11, so we know there are more projects on the next page.

Yes I did something similar to what you described in a separate PR in this commit:
https://github.com/shuttle-hq/shuttle/pull/1320/commits/25b382d608de82f65ce8d3162ccf19a6423afd5c

However I was instructed to change it to just return a record count vs a bool for the next page so I made that change and that PR still remains open here:
https://github.com/shuttle-hq/shuttle/pull/1320

However the changes are breaking so I made this that can be merged without breaking.

Reply:
Oh I just reread this as I was checking up on the PR and I realized you meant doing the limit + 1 on the client-side, I can definitely change it to do that and as you mentioned it's 100% accurate whereas this can falsely hint at a next page, I was just doing it on he server-side not the client-side which caused it to be breaking.

Reply:
Yes, sorry for not making that clear! :smile: It would be great if you could implement this and resolve the conflicts, then I think we can get this merged for the next release.